app/liquid/liquid_renderer.rb

```diff
@@ -26,7 +26,7 @@ def default_markup
   end
 
   def data
-    @data ||= Plugins.data_for_view(@page, {form_values: @member.try(:attributes)}).
+    @data ||= Plugins.data_for_view(@page, {form_values: @member.try(:attributes), url_params: @url_params}).
```

Comment:
only thing that I'm a little hesitant about is passing all the params to liquid data rather than just the relevant key / value. it just seems like it would be clearer to pass `donation_band: @url_params[:donation_band]` here and then it can be directly accessed later, not in a double hash. but it's really up to you if you want to change it